src/XMLHttpRequest/XMLHttpRequest/createXMLHttpRequestOverride.ts

```diff
@@ -291,7 +291,7 @@ export const createXMLHttpRequestOverride = (
             originalRequest.open(
               this.method,
               this.url,
-              this.async,
+              typeof this.async === 'undefined' ? true : this.async,
```

Comment:
Consider using null coalescing operator instead:

```js
this.async ?? true
```

This will respect `true` and `false`, and fallback to the `true` only if the value of `this.async` is `undefined`/`null`.

Reply:
Ah, yes that's a nice operator :)